test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertFrom-Json.Tests.ps1

```diff
@@ -7,6 +7,29 @@ Describe 'ConvertFrom-Json' -tags "CI" {
             @{ AsHashtable = $true  }
             @{ AsHashtable = $false }
         )
+
+        $testCasesJsonDepthWithAndWithoutAsHashtableSwitch = @(
+            @{ Depth = 2;    AsHashtable = $true  }
+            @{ Depth = 2;    AsHashtable = $false }
+            @{ Depth = 200;  AsHashtable = $true  }
+            @{ Depth = 200;  AsHashtable = $false }
+            @{ Depth = 2000; AsHashtable = $true  }
```

Comment:
How long does this take to run?  If longer than 2 secs, we should have them as Feature tests instead of CI.